cmd/generate-config/incus_doc.go

```diff
@@ -38,8 +38,15 @@ type doc struct {
 // sortConfigKeys alphabetically sorts the entries by key (config option key) within each config group in an entity.
 func sortConfigKeys(projectEntries map[string]any) {
 	for _, entityValue := range projectEntries {
-		for _, groupValue := range entityValue.(map[string]any) {
-			configEntries := groupValue.(map[string]any)["keys"].([]any)
+		groupValues, ok := entityValue.(map[string]any)
+		if !ok {
+			continue
+		}
+		for _, groupValue := range groupValues {
+			configEntries, gok := groupValue.(map[string]any)["keys"].([]any)
+			if !gok {
+				continue
+			}
```

Comment:
Line break needed after this line.

Reply:
64a89a410249a7f8f65aef84d3f391d4fc211949